File: test/java/dsa/MyPriorityQueueTest.java
package dsa;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.AfterAll;
import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeAll;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class MyPriorityQueueTest {
    MyPriorityQueue<Integer> myPriorityQueue;

    @BeforeAll
    static void initAll() {

    }

    @BeforeEach
    void init() {
        myPriorityQueue = new MyPriorityQueue<>();
        myPriorityQueue.clear();
    }

    @Test
    void test() {
        assertTrue(true, "Not true");
    }

    @AfterEach
    void tearDown() {

    }

    @AfterAll
    static void tearDownAll() {

    }
}

File: test/python/dsa/mypriorityqueuetest.py
import sys
import unittest

sys.path.append("src/python")
from dsa.mypriorityqueue import MyPriorityQueue


class MyPriorityQueueTest(unittest.TestCase):
    @classmethod
    def setUpClass(cls) -> None:
        return super().setUpClass()

    def setUp(self) -> None:
        self.myPriorityQueue = MyPriorityQueue[int]()
        self.myPriorityQueue.clear()

    def test(self):
        self.assertTrue(True, "not true")

    def tearDown(self) -> None:
        return super().tearDown()

    @classmethod
    def tearDownClass(cls) -> None:
        return super().tearDownClass()


if __name__ == "__main__":
    unittest.main()
